### PR TITLE
[FE] : Admin login and Register

### PR DESCRIPTION
**Pull request by kuldeepsaini2003s**
[FE] : Admin Register And Login UI

### CODE REVIEW

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    console.log(userData);
+    // Check if OTP and verifyOTP are equal
+    if (OTP != otpVerify) {
+      alert("OTP verification failed");
+      return;
+    }
+
+    try {
+      const response = await fetch(
+        "https://zummit-kefo.onrender.com/api/users/register",
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/Login-reg.js:34]: Can u please change this link ? It's a admin not a user and It's not a updated .
https://zummit-chandan.onrender.com/api/admin/register

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+      navigate("/userdashboard");
+    } catch (error) {
+      console.error("Error:", error);
+    }
+  };
+
+  const loginUser = async (loginData) => {
+    try {
+      const response = await fetch(
+        "https://zummit-kefo.onrender.com/api/users/login",
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/Login-reg.js:66]: Can u please change this link ? It's a admin not a user and It's not a updated .
https://zummit-chandan.onrender.com/api/admin/login

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+        }
+      );
+
+      if (!response.ok) {
+        throw new Error("Login failed");
+      }
+
+      const data = await response.json();
+      console.log(data);
+      dispatch(addUser(data.newUser));
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/Login-reg.js:84]: We have to create redux for the admin as well. It's user redux

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+    console.log(userData);
+    // Check if OTP and verifyOTP are equal
+    if (OTP != otpVerify) {
+      alert("OTP verification failed");
+      return;
+    }
+
+    try {
+      const response = await fetch(
+        "https://zummit-kefo.onrender.com/api/users/register",
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:34]: Can u please change this link ? It's a admin not a user and It's not a updated .
https://zummit-chandan.onrender.com/api/admin/register

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+      }
+    } catch (error) {
+      console.error("Error:", error);
+    }
+  };
+
+  const TherapistLogin = async (loginData) => {
+    try {
+      const response = await fetch(
+        "http://localhost:4000/api/therapist/loginTherapist",
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:103]: Can u please change this link ? It's a admin not a user and It's not a updated .
https://zummit-chandan.onrender.com/api/admin/register

No local host

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+  //token check karo reload kee baad
+  const checkForToken = () => {
+    const token = localStorage.getItem("token");
+    if (token) {
+      // Dispatch an action to set the user state with the token
+      dispatch(addUser({ token }));
+    }
+  };
+
+  //re-render kee baad call karo
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:147]: delete this , if it is not in use

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js [hunk 7]
```diff
@@ -0,0 +1,373 @@
...
+    } catch (error) {
+      console.error("Error:", error);
+    }
+  };
+
+  //token check karo reload kee baad
+  const checkForToken = () => {
+    const token = localStorage.getItem("token");
+    if (token) {
+      // Dispatch an action to set the user state with the token
```
Comment (chandan24042001s) [inline frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:142]: Remove extra comments. Use meningful resuable variables

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js (hunk 1 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/Login-reg.js:34]: addressed

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js (hunk 2 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/Login-reg.js:66]: fixed

File: frontend/src/components/Admin/Register_Loigin/Login-reg.js (hunk 3 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/Login-reg.js:84]: fixed

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js (hunk 4 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:34]: fixed

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js (hunk 5 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:103]: fixed

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js (hunk 6 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:147]: fixed

File: frontend/src/components/Admin/Register_Loigin/SignUp-reg.js (hunk 7 above)
Comment (kuldeepsaini2003s) [reply frontend/src/components/Admin/Register_Loigin/SignUp-reg.js:142]: Okie

**Pull request merged**